src/authUtils.ts

```diff
@@ -0,0 +1,32 @@
+import { AccessToken } from './AccessToken';
+import { VideoGrant } from './grants';
+
+/**
+ * Utilities to handle authentication
+ */
+export class authUtils {
```

Comment:
Sorry, I misunderstood your question earlier. If this is a class, then it'd be better to make this a super class encapsulating it. class names are always `CamelCase`. What do you think about `class ServiceBase`

where other Services simply extend off of it?

```suggestion
export class authUtils {
```

Reply:
As a mostly Go developer, I do have opinions about subclassing 😊. But I'll trust the folks who actually write js.